File: Java/VisorCuentas/src/main/java/modelo/Cuenta.java
package modelo;



public class Cuenta
{
    String numeroCuenta;
    String titular;
    String fechaApertura;
    String saldo;

    public Cuenta(String fnumeroCuenta, String ftitutlar, String ffechaApertura, String fsaldo)
    {
        this.numeroCuenta = fnumeroCuenta;
        this.titular = ftitutlar;
        this.fechaApertura = ffechaApertura;
        this.saldo = fsaldo;
    }

}

File: CS/Visor Cuentas/VisorWPF/Cuenta.cs
using System;


namespace VisorWPF
{
    class Cuenta
    {
        public Cuenta(string fnumero, string ftitular, string ffecha, double fsaldo, string fnacionalidad)
        {
            Numero = fnumero;
            Titular = ftitular;
            Fecha = ffecha;
            Saldo = fsaldo;
            Nacionalidad = fnacionalidad;
        }

        string Numero { get; set; }

        string Titular { get; set; }

        string Fecha { get; set; }

        double Saldo { get; set; }

        string Nacionalidad { get; set; }
    }
}